library/src/options.cxx

```diff
@@ -119,7 +119,9 @@ F3D_DECL_TYPE(int);
 F3D_DECL_TYPE(double);
 F3D_DECL_TYPE(f3d::ratio_t);
 F3D_DECL_TYPE(std::string);
+F3D_DECL_TYPE(std::vector<int>);
```

Comment:
would we `F3D_DECL_TYPE` all missing vectors (bool, ratio_t)?

Reply:
indeed better.